testsuite/tests/conftest.py

```diff
@@ -283,6 +282,14 @@ def account_password():
     return '123456'
 
 
+@backoff.on_exception(backoff.fibo, errors.ApiClientError, 8, jitter=None)
+def call_account(custom_account, params):
```

Comment:
a) maybe I would prefer this to be an inner function inside `def account` to hide it as "implementation detail"
b) should this be rather in `def custom_account` with new parameter `wait` this time by default set to `True`?
c) (just a comment) this catches `ApiClientError` to broadly, initially I was thinking just about checking the presence of 409 status, however as it is it may also solve the other issues raised during custom tenant, so keep it as it is. On the other hand all this will slowdown significantly the execution of all tests if something will be wrong with 3scale (and this will raise an error every time due to that). This was the reason I wanted it as a separate function to be called before the tests (now recalled the idea). Keep it as it is for now, but if we will hit that problem (and in such case it will be clearly visible), this will have to be reimplemented

Reply:
a) ack
b) ack
c) I slightly disagree with this, the test execution slows down only if there are issues in 3scale API availability, but if this is the case the execution is slow or failed in any case.